### PR TITLE
#3 Create Bottom Navigation with 3 Fragments: created bottom nav with…

### PR DESCRIPTION
**Pull request by MorjanDaniel**
… 3 fragments

### CODE REVIEW

File: app/src/main/java/com/summer/internship/tvtracker/PopularFragment.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.summer.internship.tvtracker
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.summer.internship.tvtracker.databinding.FragmentPopularBinding
+
+private lateinit var binding: FragmentPopularBinding
```
Comment (pmiclynx) [inline app/src/main/java/com/summer/internship/tvtracker/PopularFragment.kt]: keep the binding inside the PopularFragment class and also give me an explanation why you put it outside of it

File: app/src/main/java/com/summer/internship/tvtracker/TopRatedFragment.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.summer.internship.tvtracker
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.summer.internship.tvtracker.databinding.FragmentTopRatedBinding
+
+private lateinit var binding: FragmentTopRatedBinding
```
Comment (pmiclynx) [inline app/src/main/java/com/summer/internship/tvtracker/TopRatedFragment.kt]: same here but without explanation :))

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -6,14 +6,27 @@
...
     tools:context=".MainActivity">
 
-    <TextView
-        android:id="@+id/TextViewWellcomeText"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        app:layout_constraintBottom_toBottomOf="parent"
+    <fragment
+        android:id="@+id/MainContainer"
```
Comment (pmiclynx) [inline app/src/main/res/layout/activity_main.xml]: take care of the ids name as well, somewhere the id starts with the uppercase and in other places with lowercase. The convention for this is with lowercase. Please make sure you apply this convention for every id in this project

**Review by pmiclynx — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/summer/internship/tvtracker/PopularFragment.kt (hunk 1 above)
Comment (MorjanDaniel) [reply app/src/main/java/com/summer/internship/tvtracker/PopularFragment.kt]: i didn't realise i put it outside ..sorry :)))

Comment (pmiclynx) [reply app/src/main/java/com/summer/internship/tvtracker/PopularFragment.kt]: that's why code review was made for

**Review by pmiclynx — APPROVED** [review]

**Pull request merged**